src/main/java/domain/GamePlayer.java

```diff
@@ -0,0 +1,88 @@
+package domain;
+
+import io.Printer;
+import io.Receiver;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GamePlayer {
+    Printer printer;
+    Receiver receiver;
+    Generator generator;
+
+    public GamePlayer() {
+        this.printer = new Printer();
+        this.receiver = new Receiver();
+        this.generator = new Generator(10);
+    }
+
+    private final int WINNER_CONDITION = 4;
+
+    public void judgeToMove(Car car, int randomNumber) {
+        if (randomNumber >= WINNER_CONDITION)
+            car.moveForward();
+    }
+
+    public Car[] makeArrayAfterGettingName() {
+        printer.printGeneralMessage("INPUT_NAMEOFCAR");
+        String[] listOfName = receiver.receiveName();
+
+        Car[] cars = new Car[listOfName.length];
+        for (int i = 0; i < listOfName.length; i++)
+            cars[i] = new Car(listOfName[i]);
+
+        return cars;
+    }
+
+    public int makeCountAfterGettingNumber() {
+        printer.printGeneralMessage("INPUT_COUNT");
+        return receiver.receiveNumber();
+    }
+
+    public void launchAllRound(Car[] cars, int countRound) {
+        for (int i = 0; i < countRound; i++) {
+            for (Car j : cars) {
+                judgeToMove(j, generator.generateRandomNumber());
+                printer.printProgressWithSymbol(j.getName(), j.getPosition());
+            }
+            printer.printGeneralMessage("DEFAULT_SPACE");
+
+            try {
+                Thread.sleep(500);
+            } catch (InterruptedException e) {
+                e.printStackTrace();
+            }
+        }
+    }
+
+    public List<Car> checkWhoIsWinner(Car[] cars) {
+        List<Car> result = new ArrayList<>();
```

Comment:
메소드에서 리턴할 결과를 ret이나 result로 하는 것보다 
메소드의 역할을 보여주는 명확한 네이밍을 지정해주면 좋지 않을까요?

```suggestion
        List<Car> winners = new ArrayList<>();
```